feature-libs/organization/account-summary/components/account-summary/list/account-summary-list.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import {
+  ItemService,
+  ListService,
+  UnitListComponent,
+} from '@spartacus/organization/administration/components';
+import { AccountSummaryItemService } from '../../services/account-summary-item.service';
+import { AccountSummaryUnitListService } from '../../services/account-summary-unit-list.service';
+@Component({
+  selector: 'cx-account-summary-list',
+  templateUrl: './account-summary-list.component.html',
+  host: { class: 'content-wrapper' },
```

Comment:
do we need this `host: { class: 'content-wrapper' }`?

Reply:
not needed